inspire_disambiguation/api.py

```diff
@@ -47,11 +49,22 @@ def train_and_save_ethnicity_model(load_data_path, save_model_path):
         load_data_path (str): Full path to training data for ethnicity estimator.
         save_model_path (str): Full path where trained ethnicity model will be saved.
     """
+    _start = datetime.now()
```

Comment:
I would go for ```start_time``` without the _

Reply:
I prefer to leave _ for all this `time/debugging` stuff, it's easier to find it later and remove if we wish to do so.
PEP8: single_leading_underscore: weak "internal use" indicator

Reply:
I'll agree with @monaawi it make it more readable without ``_``

Reply:
:+1: